doc/sphinx/addendum/program.rst

```diff
@@ -295,6 +295,14 @@ optional tactic is replaced by the default one if not specified.
 
    Start the proof of the next unsolved obligation.
 
+.. cmd:: Final Obligation {? with @ltac_expr }
+
+   Like :cmd:`Next Obligation`, start the proof of the next unsolved
+   obligation. Additionally, at :cmd:`Qed` time after the
+   automatic solver has run on any remaining obligations, Coq checks
+   that no obligations remain for the given :token:`ident` when
+   provided and globally in the current module when none is provided.
```

Comment:
```suggestion
   that no obligations remain for the given :token:`ident` (when
   provided) and otherwise throughout the current module.
```

"globally in the current module" - "globally" somewhat contradicts "in the current module", so not the best wording

Would you also describe `ltac_expr`, e.g. add some words like ```using the given :token:`ltac_expr` or the default one.```

Reply:
I don't know what the ltac_expr does, but it's the same as Next Obligation and therefore not needed for this PR